apps/storybook/src/feedback/Banner.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Flex, Link, Text } from "@optiaxiom/react";
+import {
+  Banner,
+  BannerDescription,
+  BannerTitle,
+} from "@optiaxiom/react/unstable";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  component: Banner,
+} as Meta<typeof Banner>;
+
+type Story = StoryObj<typeof Banner>;
+
+export const Basic: Story = {
+  args: {
+    children: (
+      <>
+        <BannerDescription flexDirection="row">
+          <Text flex="1">
+            One or more of your social channel access tokens have expired.
+          </Text>
+
+          <Text>
+            <Link colorScheme="neutral" href="data:,">
+              Sync now
+            </Link>
+          </Text>
+        </BannerDescription>
+      </>
+    ),
+    colorScheme: "danger",
+    onClose: () => {},
+  },
+  parameters: {
+    layout: "padded",
+  },
```

Comment:
Why do we need it?

Reply:
to show the banner at the top of the screen in a banner form

Reply:
Then we need to add it in the `Meta`, so every story has the same behaviour.

export default {
  component: Banner,
  parameters: {
    layout: "padded",
  },
} as Meta<typeof Banner>;

Reply:
These makes a lot more sense. Thanks bro!